src/test/java/CarDaoTest.java

```diff
@@ -0,0 +1,31 @@
+import cz.muni.fi.PersistenceApplicationContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceContext;
+import javax.persistence.PersistenceUnit;
+
+/**
+ * Author: Richard Hrmo
+ */
+
+@ContextConfiguration(classes = PersistenceApplicationContext.class)
+public class CarDaoTest {
```

Comment:
Pls move tests outside of main directory.. I wrote on messenger dir structure of tests